pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/VenueImageUploaderModal.tsx

```diff
@@ -45,6 +45,7 @@ export const VenueImageUploaderModal = ({
   const [croppingRect, setCroppingRect] = useState<CroppedRect>()
   const [editedImage, setEditedImage] = useState('')
   const [isUploading, setIsUploading] = useState(false)
+  const [scale, setScale] = useState(1)
```

Comment:
Afin de bien distingué les deux valeurs entre `scale` ici et `scale` dans `ImageEditor`. Ici cela peut etre `editorInitialScale` ou autre.
```suggestion
  const [editorInitialScale, setEditorInitialScale = useState(1)
```